rollup/ui5-resolve.js

```diff
@@ -0,0 +1,118 @@
+const path = require("path");
+const fs = require("fs");
+const stream = require("stream");
+const { promisify } = require("util");
+const pipe = promisify(stream.pipeline);
+
+const convertAMDtoES6 = require("@buxlabs/amd-to-es6");
+const amdextract = require("amdextract");
+
+const log = require("@ui5/logger").getLogger("rollup-plugin-ui5-resolve");
```

Comment:
maybe rename it to `rollup-plugin-ui5-resolve-internal` since I might publish the `rollup-plugin-ui5-resolve` as a separate project to not collide - then you can also switch to the official one...

Reply:
Renamed to something like "rollup-plugin-ui5-resolve-task-adaptation" to distinguish better.